azure_li_services/units/system_setup.py

```diff
@@ -258,6 +259,17 @@ def set_stonith_service(config):
         )
 
 
+def load_stonith_needed_modules():
```

Comment:
Naming issue. The method is named "load" and as such it has a side effect, it writes to disk. That would not be expected based on the name of the implementation. So if we want to load and make it boot persistent in the same implementation we need a new name `enable_extra_kernel_module(module_name)` for example would be a generic implementation. That then basically does what you have currently implemented.